tests/tantivy_test.py

```diff
@@ -1,9 +1,15 @@
 import json
 import tantivy
 
+import pytest
+
 
 class TestClass(object):
-    def test_simple_search(self):
+
+    @classmethod
+    def setup_class(cls):
```

Comment:
The usual way to achieve this is to setup a pytest fixture. That way you could also have multiple index fixtures for differing schemas for example. More info [here](https://docs.pytest.org/en/latest/fixture.html).